src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/BackupUtilities.cs

```diff
@@ -136,6 +147,109 @@ public BackupConfigInfo GetBackupConfigInfo(string databaseName)
         }
 
         /// <summary>
+        /// Execute backup
+        /// </summary>
+        public void PerformBackup()
+        {
+            this.backup = new Backup();
+            this.SetBackupProps();
+            backup.Database = this.backupInfo.DatabaseName;
+            backup.Action = this.backupActionType;
+            backup.Incremental = this.IsBackupIncremental;
+            if (backup.Action == BackupActionType.Files)
+            {
+                IDictionaryEnumerator IEnum = this.backupInfo.SelectedFileGroup.GetEnumerator();
```

Comment:
>IEnum [](start = 38, length = 5)

variables should use camelCase names...also this name isn't particularly descriptive.   Maybe something like fileGroupEnum or something like that.